modules/storage-persistent-storage-azure-file-pvc.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage/persistent-storage-azure-file.adoc
+
+[id="create-azure-file-pvc_{context}"]
+= Create the Azure File PersistentVolumeClaim
+
+A PersistentVolumeClaim represents a request for storage by a user or
```

Comment:
I'd say "Create a PersistentVolumeClaim, which is a storage request, so that Pods can access to the underlying volumes."